packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -161,6 +161,17 @@ describe('plugin-meetings', () => {
       assert.isFalse(audio.isSelf());
     });
 
+    describe.only('#isLocallyMuted()', () => {
```

Comment:
`.only` needs to be removed

Reply:
Sorry. I missed to remove it while pushing. Thanks for noticing. Changed now.